src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -83,21 +89,51 @@ function CatalogueCard(props: CatalogueCardProps) {
           <IconButton
             onClick={(event) => {
               event.preventDefault();
-              onChangeOpenEditDialog(catalogueCategory);
+              setAnchorEl(event.currentTarget);
+              setMenuOpen(true);
             }}
-            aria-label={`edit ${catalogueCategory.name} catalogue category button`}
+            aria-label={`actions ${catalogueCategory.name} catalogue category button`}
           >
-            <EditIcon />
+            <MoreHorizIcon />
           </IconButton>
-          <IconButton
-            onClick={(event) => {
-              event.preventDefault();
-              onChangeOpenDeleteDialog(catalogueCategory);
+          <Menu
+            anchorEl={anchorEl}
+            open={menuOpen}
+            onClose={(event) => {
+              setMenuOpen(false);
```

Comment:
Could this line also be covered? I assume it would just need a unit test that opens and closes the menu.